src/client/cli/cmd/get.cpp

```diff
@@ -67,34 +65,71 @@ QString cmd::Get::short_help() const
 
 QString cmd::Get::description() const
 {
-    auto desc = QStringLiteral("Get the configuration setting corresponding to the given key.");
-    return desc + "\n\n" + describe_settings_keys();
+    auto desc = QStringLiteral("Get the configuration setting(s) corresponding to the given key(s), or all settings if "
+                               "no key is specified.");
+    return desc + "\n\n" + describe_common_settings_keys();
 }
 
 mp::ParseCode cmd::Get::parse_args(mp::ArgParser* parser)
 {
-    parser->addPositionalArgument("key", "Path to the setting whose configured value should be obtained.", "<key>");
+    parser->addPositionalArgument(
+        "arg", "Setting(s) key(s), i.e. path(s) to the setting(s) whose configured value(s) should be obtained.",
+        "[<arg> ...]");
 
     QCommandLineOption raw_option("raw", "Output in raw format. For now, this affects only the representation of empty "
                                          "values (i.e. \"\" instead of \"<empty>\").");
+    QCommandLineOption keys_option("keys", "List available settings keys. This outputs the whole list of currently "
+                                           "available settings keys, or a matching subset when any <arg> are "
+                                           "provided");
```

Comment:
And here, as you say it's a bit useless (well, it lets you validate the key), but what's not supported shouldn't be in help text.

Reply:
Let me know if it's OK now or if there are further changes you'd like to see.